app/src/main/java/com/duycuong/weather/ui/screen/main/MainViewPagerAdapter.java

```diff
@@ -0,0 +1,48 @@
+package com.duycuong.weather.ui.screen.main;
+
+import android.content.Context;
+import android.content.res.Resources;
```

Comment:
ERROR: Unused import: android.content.res.Resources. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports